src/ui/states/state-present-offer.svelte

```diff
@@ -30,34 +37,71 @@
 
     {#if isSubscription}
             <span class="rcb-product-price">
+
                 {#if subscriptionTrial?.periodDuration}
-                    {getTrialsLabel(subscriptionTrial.periodDuration)} free trial
+                  <Localized
```

Comment:
it's complaining about props on all of these 🤔

Reply:
Updated all of these, now all the props are optional